tests/specs/benchmarks/structarg01-spec.k

```diff
@@ -0,0 +1,128 @@
+requires "verification.k"
```

Comment:
I assume this will change when the test changes?

Reply:
Yes.